genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/ClusterTask.java

```diff
@@ -44,14 +44,18 @@
 public class ClusterTask extends GenieBaseTask {
 
     private final Timer timer;
+    private final GenieFileTransferService fts;
 
     /**
      * Constructor.
      *
      * @param registry The metrics registry to use
+     * @param fts File transfer service
      */
-    public ClusterTask(@NotNull final Registry registry) {
+    public ClusterTask(@NotNull final Registry registry,
```

Comment:
We seem to be repeating a lot of this signature in the constructors I'd move it into an abstract base class for reuse